KsApi/models/DebugData.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+public struct DebugData {
```

Comment:
Hmm just wondering if this should be a `KsApi` model... it's really just a wrapper type that we're using in the app, should perhaps just live in `Library?`